.github/workflows/lint-code-base.yml

```diff
@@ -0,0 +1,31 @@
+name: Lint Code Base
+
+on:
+  pull_request: null
```

Comment:
Is this a change in behavior? Previously, linter was supposedly `on` for PRs with a list of file patterns.

Reply:
It is on for PRs with no specific file patterns (e.g. all PRs).  The idea is it allows for the easy enablement of other linters supported by the super-linter.  e.g. scripting, json, yml, etc.